Core/Inc/NodeInfos.h

```diff
@@ -136,8 +148,10 @@ extern const ChannelLookUpEntry channellookUp[];
 //#define LAMARR_ECU_NODE_ID 5
 //#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_OX_ECU
 //#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_FUEL_ECU
-//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_PNEU
-//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_ELEC
-#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_ENGINE_ECU
+//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_PNEU_0
+#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_PNEU_1
+//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_ELEC_IOB
+//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_ELEC_BOB
+//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_ENGINE_ECU
```

Comment:
Can you reorder it so that the ENGINE_ECU? is under the fuel?
```suggestion
//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_ENGINE_ECU
//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_PNEU_0
#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_PNEU_1
//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_ELEC_IOB
//#define LAMARR_ECU_NODE_ID NODE_ID_LAMARR_GSE_ELEC_BOB

```